pagopa/io-app
# fix(IT Wallet): [SIW-1734] Add "back to top" in the wallet home screen

## Short description
This PR adds the ability to return to the top of the wallet screen when tapping on the wallet tab navigation icon

## List of changes proposed in this pull request
- Added `useTabItemPressWhenScreenActive` handler in `WalletHomeScreen`

## How to test
Navigate to the Wallet home screen, scroll to the bottom, tap on the tab navigation icon and check that the scroll view scrolls back to the top

## Preview

https://github.com/user-attachments/assets/f9194597-cadd-4da3-a583-8e34c4e56a54
